src/main/java/com/lpvs/service/LPVSGitHubService.java

```diff
@@ -362,7 +338,7 @@ public void commentResults(
                     webhookConfig.getHeadCommitSHA(),
                     GHCommitState.SUCCESS,
                     null,
-                    "No license issue detected",
+                    "No license issue(s) detected",
```

Comment:
In the above code, there is message:  "No license issue detected" (line 335). With this message, they are a little bit different. But in the above code, there are messages:
"Potential license problem(s) detected" (line 323)
"Potential license problem(s) detected" (line 329)
So maybe it will be more consistency to use the same style for all messages: "No license(s) issue detected"

Also, another question, is it normal that we use "problem" in one case and "issue" in another?

Reply:
Thank you for the comment. I've updated all messages.